repository/src/main/java/org/alfresco/repo/content/AbstractRoutingContentStore.java

```diff
@@ -420,23 +421,71 @@ public boolean delete(String contentUrl) throws ContentIOException
         return deleted;
     }
 
+    /**
+     * {@inheritDoc}
+     */
     @Override
     @Experimental
     public Map<String, String> getStorageProperties(String contentUrl) {

```

Comment:
Formatting { new line I can see we have 2 formatting styles in this file, some were not caught in previous PRs. This project looks like it used { on new lines.